### ISSUE TITLE
procure Macs for FY22

### ISSUE THREAD

**Issue by afeld** [18F/tts-tech-portfolio#1460]
## Background Information

https://trello.com/c/b2XQb5AT/264-ensure-theres-adequate-inventory-for-tts-through-fy22

## Implementation Steps

See Trello card.
- [x] submit package through EASI

## Acceptance Criteria

- [x] We have adequate Mac inventory for projected FY22 hires

**Comment by afeld** [comment]
This isn't exactly `expedited`, as it's been percolating in the background for a while. Just realized I wasn't tracking it on the board, so adding to reflect reality.

**Comment by afeld** [comment]
Set up meeting tomorrow (7/22) to try and figure out financing piece.

**Comment by afeld** [comment]
Waiting on EASi access.

**Comment by JJediny** [comment]
Purchase Request for 60 units was completed and delivery confirmed as of 2/2022. Will create a new issue for the FY23 need of 300 additional units
